core/app.py

```diff
@@ -31,10 +31,13 @@ def init_db():
 
 
 def init_cache():
+    redis_password = config.secrets.server('redis.password')
+    redis_password = None if redis_password == 'password' else redis_password
```

Comment:
Can you change the logic here to make the password None if the password field doesn't exist in the config? It feels a bit weird that the way to not use a password is to have the explicit string "password".

Reply:
True. I'll change it.